extensions/android/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+import("//build/config/android/rules.gni")
+
+android_library("xwalk_core_extensions_java") {
+  deps = [
+    "//base:base_java",
+    "//content/public/android:content_java",
+  ]
+  java_files = [
+    "java/src/org/xwalk/core/internal/extensions/XWalkExtensionAndroid.java",
+    "java/src/org/xwalk/core/internal/extensions/XWalkNativeExtensionLoaderAndroid.java",
+  ]
+}
+
+generate_jni("xwalk_core_extensions_native_jni") {
+  jni_package = rebase_path(
```

Comment:
In GYP, all the jni headers are generated into `$root_out_dir/jni/`
If I set "jni_package=xwalk" here,  the jni headers will be generated into `$target_out_dir/xwalk/jni/`, so I use a rebase_path to make sure all the jni headers are generated same path as in GYP.